sovtoken/sovtoken/test/req_handlers/utxo_cache_batch_handler/test_token_batch_handler_commit_batch.py

```diff
@@ -3,13 +3,13 @@
 from sovtokenfees.static_fee_req_handler import txn_root_serializer
 
 
-def test_token_batch_handler_commit_batch(token_batch_handler, utxo_cache):
+def test_token_batch_handler_commit_batch(utxo_batch_handler, utxo_cache):
```

Comment:
A correct test name is `test_utxo_batch_handler_commit_batch`

Reply:
fixed